govuk-paas/paas-rds-broker-boshrelease
# [#168322475] Add support for Postgres 11 and MySQL 8.0

## What

Add support for Postgres 11 and MySQL 8.0 by releasing https://github.com/alphagov/paas-rds-broker/pull/107 and https://github.com/alphagov/paas-rds-broker/pull/108.

## How to review

* Check the boshrelease builds successfully;
* Deploy the updated boshrelease to a dev env and try it out.

## Who can review

Not @46bit
